pages/common/gum.md

```diff
@@ -0,0 +1,28 @@
+# gum
+
+> A tool for making glamorous shell scripts.
+> More information: <https://github.com/charmbracelet/gum>.
+
+- Interactively pick an option to print to `STDOUT`:
+
+`gum choose "{{option_1}}" "{{option_2}}" "{{option_3}}"`
+
+- Open an interactive prompt for the user to input a string with a placeholder:
+
+`gum input --placeholder "{{placeholder_string}}"`
+
+- Interactvely pick an option which will exit if "No" is chosen:
+
+`gum confirm "{{Continue?}}" --default=false --affirmative "{{Yes}}" --negative "{{No}}" {{&& : || exit 1}}`
+
+- Show a spinner while a command is taking place with text alongside:
+
+`gum spin --spinner {{dot|line|minidot|jump|pulse|points|globe|moon|monkey|meter|hamburger}} --title "{{loading...}}" -- {{command}}`
```

Comment:
```suggestion
`gum spin --spinner {{dot|line|minidot|jump|pulse|...}} --title "{{loading...}}" -- {{command}}`
```

Let's use `...` when there are too many alternatives to list.

Reply:
Yeah, This seems better. It would fix the [issue](https://github.com/tldr-pages/tldr/pull/8314#discussion_r939615534).

Reply:
This could work, although as the user I have no idea of what the other options are and since they aren't obvious like when we use {[file1 file2 ...}}, I'd prefer to keep them.

Reply:
In my opinion, our aim is to help the `user`; and if the options are not absolutely obvious and can fit on an example, we should have them.
Although, there is may be a balance that we folks might want to strike where we do not make it too verbose :)

Reply:
This example with all possible options is exactly as long as the previous one. I think we can keep them all.